src/agent/script/win64/libfuzzer-coverage/DumpCounters.js

```diff
@@ -20,15 +20,16 @@ function readU8Array(addr, len) {
 }
 
 // For future research: Other tables of interest in MSVC 16.8
-// _Sancov8bitUsed - __sancov$8bitCountersStart & __sancov$8bitCountersEnd
 // _SancovPcGuardUsed - __sancov$TracePCGuardStart & __sancov$TracePCGuardEnd
 // _SancovPcTableUsed - __sancov$PCTableStart & __sancov$PCTableEnd
 
 function findCounterSymbols(exe) {
     var symbols = [
-        { name: "LLVM 10", start: "__start___sancov_cntrs", end: "__stop___sancov_cntrs" },
         { name: "MSVC 16.8", start: "__sancov$BoolFlagStart", end: "__sancov$BoolFlagEnd" },
+        { name: "MSVC 16.8", start: "__sancov$8bitCountersStart", end: "__sancov$8bitCountersEnd" },
```

Comment:
Can we differentiate between MSVC 17.8 that used BoolFlagStart and 8bitCountersStart by name here?  It  will help in future debugging.

Reply:
Done -- happy to change the names if you'd prefer them in a different format. Mapped them to the compiler args used to generate them:
```
inline-8bit-counters -> "8bit counters"
inline-bool-flag -> "bool flag"
```